src/linker/Linker/TypeReferenceWalker.cs

```diff
@@ -16,21 +16,20 @@ abstract class TypeReferenceWalker
 	{
 		protected readonly AssemblyDefinition assembly;
 
-		HashSet<TypeReference>? visited;
-		protected HashSet<TypeReference> Visited => visited ?? throw new InvalidOperationException ();
+		protected HashSet<TypeReference> Visited { get; init; } = new HashSet<TypeReference> ();
```

Comment:
Why does it need to be init?

Reply:
It doesn't - it just "reads" right - I'll change it.